x/programs/rust/wasmlanche-sdk/src/program.rs

```diff
@@ -47,10 +48,10 @@ impl<K> Program<K> {
     /// Returns a [`StateError`] if the call fails.
     /// # Safety
     /// The caller must ensure that `function_name` + `args` point to valid memory locations.
```

Comment:
🤔 this is out of date. Will create an issue to go and make sure all the docs are up to date and maybe add example usage everywhere.